mongodb/driver/query/construct.xml

```diff
@@ -42,7 +42,7 @@
          <row>
           <entry>limit</entry>
           <entry><type>integer</type></entry>
-          <entry>Client side maximum number of documents to return</entry>
+          <entry>Maximum number of documents to return to the client</entry>
```

Comment:
If the driver specifies a limit, then we specify `numberToReturn` on the OP_QUERY. I don't believe the driver would ever request results back from the server and then not surface them to the user. That's just wasteful, and we can avoid with some math around limit and batchSize. Is that incorrect?